app/src/main/res/values/styles.xml

```diff
@@ -8,7 +8,6 @@
     <item name="colorPrimaryDark">@color/bpkSkyBlueShade01</item>
     <item name="colorAccent">@color/bpkMonteverde</item>
     <item name="android:windowBackground">@color/bpkBackground</item>
-    <item name="android:textAppearance">?bpkTextBaseAppearance</item>
```

Comment:
This was causing a bunch of issues that were very difficult to resolve as this text appearance wasn't resolvable. This is also not set in the app theme at the moment, so we shouldn't rely on it being there.